### PR TITLE
docs: update feature added in AWSIM-Labs

### PR DESCRIPTION
**Pull request by mitsudome-r**
## Description
The current table makes it look like AWSIM is worse than AWSIM-Labs with a lot of `X`, but I don't think it is fair to make such comparisons if you are making modification on top of fork.

I would like to modify the documentation to make sure that we're not being disrespectful to upstream developers.

### Related links
N/A

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by mitsudome-r** [comment]
NOTE from the discussion with Fatih: 
We are planning to move this table to Autoware Documentation. There we will have both AWSIM-Labs and TIER IV's AWSIM team to update the table to show the difference between the two.

We can merge this PR once we create the table in Autoware Documentation.

File: docs/index.md [hunk 1]
```diff
@@ -17,20 +17,6 @@ This is a fork of [TIER IV's AWSIM](https://github.com/tier4/AWSIM).
 - [Open sourced](https://github.com/autowarefoundation/AWSIM-Labs)
 - Made with [Unity](https://unity.com/)
```
Comment (xmfcx) [inline docs/index.md:19]:
```suggestion

### Feature differences from the TIER IV/AWSIM

See [Feature differences from the TIER IV/AWSIM in the Autoware Documentation](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial/#feature-differences-from-the-awsim-and-awsim-labs).
```

**Review by xmfcx — CHANGES_REQUESTED** [review]

**Comment by xmfcx** [comment]
You also need to update the [README.md](https://github.com/autowarefoundation/AWSIM-Labs?tab=readme-ov-file#feature-differences-from-the-tier-ivawsim) section with the new text:

```
See [Feature differences from the TIER IV/AWSIM in the Autoware Documentation](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial/#feature-differences-from-the-awsim-and-awsim-labs).
```

**Comment by xmfcx** [comment]
@mozhoku could you take over this task?

File: docs/index.md [hunk 2]
```diff
@@ -17,20 +17,9 @@ This is a fork of [TIER IV's AWSIM](https://github.com/tier4/AWSIM).
...
-| Rendering Pipeline                        | HDRP             | URP                   |
-| Unity Version                             | Unity 2021.1.7f1 | Unity LTS 2022.3.21f1 |
-| Resource usage                            | Heavy            | Light                 |
-| Can reset vehicle position on runtime     | ❌                | ✅                     |
-| Multiple scene and vehicle setup          | ❌                | ✅                     |
-| Multi-lidars are enabled by default       | ❌                | ✅                     |
-| CI for build                              | ❌                | ✅                     |
-| CI for documentation generation within PR | ❌                | ✅                     |
-
+See [Feature differences from the TIER IV/AWSIM in the Autoware Documentation](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial/#feature-differences-from-the-awsim-and-awsim-labs).
```
Comment (xmfcx) [inline docs/index.md:22]:
```suggestion
See [Feature differences from the TIER IV/AWSIM in the Autoware Documentation](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial/#feature-differences-from-the-awsim-and-awsim-labs).

```

**Comment by xmfcx** [comment]
Closing for:
- https://github.com/autowarefoundation/AWSIM-Labs/pull/98

**Pull request closed without merging**